src/query/functions/src/scalars/other.rs

```diff
@@ -233,6 +236,43 @@ pub fn register(registry: &mut FunctionRegistry) {
             Value::Column(col)
         },
     );
+    registry.register_function_factory("least", |_, args_type| {
```

Comment:
The implementation of `least` and `greatest` is basically the same, can be rewritten as `array_min`.

Reply:
ok